external-crates/move/crates/move-compiler/src/command_line/compiler.rs

```diff
@@ -317,17 +316,43 @@ impl Compiler {
             Some(p) => p,
             None => VfsPath::new(PhysicalFS::new("/")),
         };
+        let mut vfs_to_original_path = HashMap::new();
 
         let targets = targets
             .into_iter()
-            .map(|p| Ok(p.to_vfs_path(&vfs_root)?))
+            .map(|p| {
+                let original = p.path;
+                let vfs = p.to_vfs_path(&vfs_root)?;
+                vfs_to_original_path.insert(Symbol::from(vfs.path.as_str()), original);
+                Ok(vfs)
+            })
             .collect::<Result<Vec<_>, anyhow::Error>>()?;
```

Comment:
Something to mention is there were some build issues in migration where two packages pointed to the same dependency via different relative paths, and those got reported as conflicting even though they were the same. I don't think we should fix that in this PR, but I suspect it may impact some of this code.